docusign_esign/models/connect_o_auth_config.py

```diff
@@ -36,7 +36,7 @@ class ConnectOAuthConfig(object):
         'authorization_server_url': 'str',
         'client_id': 'str',
         'client_secret': 'str',
-        'custom_parameters': 'str',
+        'custom_parameters': 'dict(str, str)',
```

Comment:
@RamukaRitik 
Assuming that elsewhere in the code base custom_parameters is now expected to be a dictionary

Reply:
I would suggest that this change is documented in the CHANGELOG, as it is a breaking change to the API.